### PR TITLE
chore: remove deprecated method

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

File: google-cloud-spanner/src/test/java/com/google/cloud/spanner/BatchCreateSessionsTest.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```
Comment (rahulKQL) [inline google-cloud-spanner/src/test/java/com/google/cloud/spanner/BatchCreateSessionsTest.java:21]: I don't think we should use hamcrest assert methods, Can we not use Truth here?

File: google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
```
Comment (rahulKQL) [inline google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java:22]: same here?

**Review by rahulKQL — CHANGES_REQUESTED** [review]
It would be better to add Truth dependency and use the Truth rather than hamcrest

**Comment by pmakani** [comment]
> It would be better to add Truth dependency and use the Truth rather than hamcrest

of course we can use truth but deprecated assertThat method has java docs which i am following and if googler says here you have to use truth than we can use that right ?

**Comment by rahulKQL** [comment]
I think you can directly use Truth why to wait.

**Pull request closed without merging**